### ISSUE TITLE
Dark text on dark background for broken links in storybook

### ISSUE THREAD

**Issue by jasongornall** [sourcegraph/sourcegraph#34231]
- **Sourcegraph version:** 

https://storybook.sgdev.org/?path=/story/wildcard-feedbackprompt--feedback-prompt-exampless

- **Platform information:** 
Web / Chrome / Desktop / PC

#### Steps to reproduce:

1. Click the link in dark mode and observe the black text on the dark background

#### Expected behavior:
The copy should be white

#### Actual behavior:
The copy is dark

![image](https://user-images.githubusercontent.com/6912393/164329343-82a10e00-6ef6-4564-99ca-352acbccf38c.png)

If would like immediate help on this, please email support@sourcegraph.com (you can still create the issue, but there are not [SLAs](https://about.sourcegraph.com/support/) on issues like there are for support requests).

**Comment by jasongornall** [comment]
I done into this issue as my first attempt at changing the codebase. 

It looks like this view is entirely generated [inside of storybook](https://sourcegraph.com/github.com/storybookjs/storybook/-/blob/lib/core-common/templates/base-preview-body.html?L72:49) 

All the other views work properly because of how the theme are imported directly from [BrandedStory](https://sourcegraph.com/-/editor?remote_url=git@github.com:sourcegraph/sourcegraph.git&branch=main&file=client/branded/src/components/BrandedStory.tsx&editor=VSCode&version=2.2.1&start_row=21&start_col=13&end_row=21&end_col=25&utm_campaign=vscode-extension&utm_medium=direct_traffic&utm_source=vscode-extension&utm_content=vsce-commands) But because this isn't a react component it isn't importing it on the fly when the context changes.

This must be a problem for other apps right? A brief exploration of other storybooks yielded yes and the new version has a  [hardcoded background](https://5dfcbf3012392c0020e7140b-zycekvkxin.chromatic.com/?path=/story/components-card-format-variations-standard--standardsss)

voila! I say Let's just upgrade storybook and the problem is solved.

After upgrade I see the new storybook broken link looks great!
![image](https://user-images.githubusercontent.com/6912393/164783414-f331352d-ef43-401f-ae8c-a26f92088484.png)

A quick VQA later results in some nasty react router crashes. I then learn that the new version of [storybook is compatible with react router 5](https://github.com/storybookjs/storybook/issues/16837)!!

Investigating upgrading react router they actively [advise you to wait and not upgrade.](https://reactrouter.com/docs/en/v6/upgrading/v5) until they can make it backwards compatible.


TLDR. new storybook fixes this and it needs react router to work and react router can't be upgraded yet.

**Comment by umpox** [comment]
@jasongornall

I got an email from the React Router team a couple of weeks ago saying that the migration *was* ready, with a plan on how to do it. I've emailed back to clarify what the actual status is, as the package has definitely been released but we should avoid doing so if it's not actually stable yet.

I did a quick PR here to enable https://github.com/sourcegraph/sourcegraph/pull/33835, and created an issue to migrate: https://github.com/sourcegraph/sourcegraph/issues/33834

**Comment by valerybugakov** [comment]
Fixed in https://github.com/sourcegraph/sourcegraph/pull/36437
